package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "react-movie",
+  "version": "1.0.0",
+  "description": "search movie",
+  "main": "server.js",
+  "scripts": {
+    "build": "webpack --env.production  --progress --profile --colors",
+    "start": "webpack-dev-server --env.development --watch",
+    "test": "jest",
+    "lint": "./node_modules/.bin/eslint src/**/*.js --ignore-pattern *.test.js"
```

Comment:
As I remember you can use eslint without absolute path as you used jest for example, and pls explain me why test files no need to check by eslint rules?

Reply:
Since I have not installed `eslint` as global npm module hence it was not recognizing the command directly. Also I will ass test file to eslint.